### ISSUE TITLE
[BUG] When only Chrome installed and click Record at Cursor extension want to install browsers

### ISSUE THREAD

**Issue by gemiusz** [microsoft/playwright#23906]
**Context:**
- Playwright Version: 1.35.1
- Operating System: Windows
- Node.js version: 20.3.0
- Visual Studio Code version: 1.79.2
- Playwright for VSCode extension version: 1.0.13
- Browser: Installed only Chrome
- Extra: Not downloaded/installed other browsers

Configuration (playwright.config.ts):
```TypeScript
import { defineConfig, devices } from '@playwright/test';

export default defineConfig({
  testDir: './tests',
  fullyParallel: true,
  forbidOnly: !!process.env.CI,
  retries: process.env.CI ? 2 : 0,
  workers: process.env.CI ? 1 : undefined,
  reporter: 'html',
   use: {
       trace: 'on-first-retry',
  },

  projects: [
    {
      name: 'Google Chrome',
      use: { ...devices['Desktop Chrome'], channel: 'chrome' },
    },
  ]
});

```

**Describe the bug**

I have project where want to test only against Google Chrome so I don't have installed other browsers, when using Playwright for VSCode extension and try add aditional code by ```Record at Cursor``` Playwright extension show popup to install browsers:

![image](https://github.com/microsoft/playwright/assets/30750006/1311c2f2-5927-4c39-82e7-ba5c3fa4d684)

I try to unselect all and press OK but nothing happen :/

**Comment by mxschmitt** [comment]
Record new, Record at cursor, and Pick Locator won't respect the playwright config. Since we don't know which of your Playwright config files you want to use and which project.

All of them interact with the launched Playwright browser, aka. the "Show browser" feature, if you have one already launched it will use it.

TL;DR: We recommend to run a test with Show Browser checked, and then all of the 3 features above will use the opened browser, in your case stable Chrome.

Would that work for you?

**Comment by gemiusz** [comment]
Yes, it's working as expected.

**Comment by mxschmitt** [comment]
Then closing for now since it works as expected.
